Index File Added

Nboaram opened the pull request:
=index.HTML=
- Title Added
- Table For Structure Added
- Add Button
- Search Button
- Update Button Added
- Delete Button Added
=delete.js=
- Console Log removed
--------------------------------------
- [x] Semi Colons rather than ASI
- [x] Clear Method & Variable Names
- [x] Correct use of CSS & Bootstrap
- [x] No inline Styling
- [x] External Scripts and CSS attached to HTML
- [x] Homepage called index.html
- [x] Refactored to be more efficient/readable/maintainable/scalable
-------------------------------------------
- [x] JavaScript commit doesn't overwrite index.html

Nboaram committed:
Index File Added

=index.HTML=
- Title Added
- Table For Structure Added
- Add Button
- Search Button
- Update Button Added
- Delete Button Added
=delete.js=
- Console Log removed

AChar95 reviewed (commented):
Requires styling to be moved to css

Nboaram committed:
Additional Changes

delete renamed to DeleteOwner
index styles moved to CSS
petStyle.css : table width moved into here
